gpg-insecure-passphrase-20170614.json

```diff
@@ -10,6 +10,7 @@
     }
   ],
   "tags": [
-    "gpg-incorrect-passphrase"
+    "gpg-incorrect-passphrase",

```

Comment:
looking into screenshot , agree with that it is about insecure but I didn't get why you did addition not replacement ? incorrect passphrase is different test case in my understanding .

Reply:
That is because if this PR merged before test case PR and there is case running during this period, it will failed to find the tag. But any way, I have removed it, please merge too PRs together.